cpp/CMakeLists.txt

```diff
@@ -549,6 +549,17 @@ add_library(
   src/utilities/type_checks.cpp
 )
 
+set_source_files_properties(
+  src/binaryop/binaryop.cpp
+  src/transform/transform.cpp
+  src/rolling/detail/rolling_variable_window.cu
+  src/rolling/detail/rolling_fixed_window.cu
+  src/rolling/grouped_rolling.cu
+  src/rolling/rolling.cu
+  src/jit/cache.cpp
+  PROPERTIES COMPILE_DEFINITIONS "_FILE_OFFSET_BITS=64"
```

Comment:
Can we document why we're doing this in a comment? Same for other CMake files.

Reply:
FIx'd